src/accessibility/describe.js

```diff
@@ -321,9 +320,9 @@ function _describeLabelHTML(dummyDOM, cnvId, text) {
       .querySelector(`#${cnvId + labelTableId}`)
       .insertAdjacentHTML('beforebegin', `<p id="${cnvId}${labelDescId}"></p>`);
   }
-  //update description
-  dummyDOM.querySelector('#' + cnvId + labelDescId).innerHTML = text;
-  return dummyDOM;
+  if (dummyDOM.querySelector(`#${cnvId}${labelDescId}`)) {
```

Comment:
similar questions/comments here as for _describeFallbackHTML with this return clause and the insertion of the HTML earlier on